f600233d1b154d507b782f84604bcb12/update/sys_script_include_f563b62d47152110f39f7601e36d43de.xml

```diff
@@ -37,12 +37,21 @@ HothProvisioning.prototype = {
       var clinTableFields = [
         "clin_number", "pop_start_date", "pop_end_date", "classification_level", "type", 
       ];
-      
+		var classificationMap = {
+			U: "UNCLASSIFIED",
+			S: "SECRET",
+			TS: "TOP_SECRET"
+		};
```

Comment:
servicenow tables and data will remain as they are an we'll translate here?

Reply:
Yep